adserver/models.py

```diff
@@ -552,6 +566,14 @@ def show_to_keywords(self, keywords):
         If *any* keywords are in the excluded list, it should not be shown.
         """
         keyword_set = set(keywords)
+        # Add default keywords from publisher
```

Comment:
The publisher keywords should probably just be added to the keywords list in the decision engine here: https://github.com/readthedocs/ethical-ad-server/blob/eb2b99baf24786ebca8eab7b2cc18e6918ce64c0/adserver/decisionengine/backends.py#L38